BusinessTampereTrafficMonitoring/traffic_lights/api_client.py

```diff
@@ -29,6 +28,15 @@ class TrafficLightCycle(Base):
 
 class TrafficLightAPIClient:
     def __init__(self, url: str, monitored_devices: List[str], db: str):
+        """
+        Constructor for initializing TrafficLightAPIClient class
+
+        #Parameter:
+        # argument1: url (String)
+        # argument2: list of monitored devices List[string]
```

Comment:
I would specify that the "monitored_devices" refers to intersection ids. For example "TRE401", "TRE428".

Reply:
sorry, then the naming should be changed. It does not go with the functionality.

Reply:
The reason it's called monitored devices is that the traffic light API we are using calls them devices (the end point we are using is `/deviceState`). Each intersection has one "device".